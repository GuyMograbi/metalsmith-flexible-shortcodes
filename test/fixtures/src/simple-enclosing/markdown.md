---
title: Markdown
shortcodes: true
---
This shows a simple self enclosing shortcode in markdown:

[div]

```js
$('a:empty').hide();
```

[/div]
