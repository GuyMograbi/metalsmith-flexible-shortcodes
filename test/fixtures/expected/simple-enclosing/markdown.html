<p>This shows a simple self enclosing shortcode in markdown:</p>
<div>
<pre><code class="lang-js">$(&#39;a:empty&#39;).hide();
</code></pre>
</div>
